package/pyproject.toml

```diff
@@ -79,6 +79,7 @@ extra_formats = [
     "pytng>=0.2.3",
     "gsd>3.0.0",
     "rdkit>=2020.03.1",
+    "imdclient",
```

Comment:
We almost certainly need to add a minimal version.

Reply:
Done